Steps/Education.cs

```diff
@@ -0,0 +1,62 @@
+﻿using NUnit.Framework;
+using TechTalk.SpecFlow;
+
+namespace MarsQA_1.Feature
+{
+    [Binding]
+    public class Education
+    {
+
+        MarsQA_1.SpecflowPages.Pages.EducationProfile EducationPage = new SpecflowPages.Pages.EducationProfile();
+
+        [Given(@"User enter UniversityName as ""(.*)""")]
+        public void GivenSellerEnterUniversityNameAs(string p0)
+        {
+            EducationPage.University = p0;
+        }
+
+        [Given(@"Select CountryOfUniversity as ""(.*)""")]
+        public void GivenSelectCountryOfUniversityAs(string country)
+        {

```

Comment:
Please add all these details into 1 step by consolidating all the And's into a When

And also pass as a parameter into the respective methods